.github/workflows/build.yaml

```diff
@@ -37,6 +37,65 @@ jobs:
           name: packages
           path: ${{ env.NUPKG_OUTDIR }}
 
+  test:
+    name: Test .NET ${{ matrix.dotnet-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        dotnet-version: ['6.0']
+        include:
+        - dotnet-version: '6.0'
+          install-3: false
+          display-name: '.NET 6.0'
+          framework: 'net6.0'
+          prefix: 'net6'
+          install-version: '6.0.x'
+    steps:
+    - uses: actions/checkout@v1
+    - name: Parse release version
+      run: python ./.github/scripts/get_release_version.py
+    - name: Setup ${{ matrix.display-name }}
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: ${{ matrix.install-version }}
+    - name: Setup .NET 7.0 # net7 is always required.
+      uses: actions/setup-dotnet@v1
+      if: ${{ matrix.install-version != '7.0.x' }}
+      with:
+        dotnet-version: 7.0.x
```

Comment:
Didn't you have a commit removing dot7 from the matrix? And it is still required?

Reply:
This workflow is borrowed from the Dapr .NET Client SDK; it seems to think .NET 7 is needed even when testing against earlier versions.  While I *did* remove the .NET 7 test target, for now, I kept the installation of that version just to be safe.  There's a separate task (#16) to specifically target .NET 7 which will just bring back that test target.  (It required more project changes, which would have just bloated this PR.)

Reply:
I guess it makes sense: you can test any previous version with .Net 7 but